dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1354,6 +1861,84 @@ internal class RumViewScopeTest {
         assertThat(testedScope.activeResourceScopes).isEmpty()
     }
 
+    @Test
+    fun `𝕄 wait for pending Resource 𝕎 handleEvent(StartResource) on active view`(
```

Comment:
maybe we can actually send a StopView event and show that the current view scope will not be closed in this case ?

Reply:
The problem is that sending a stopview will have a lot more interactions and will complexify this test. There are already tests that ensure that a stop view will wait until all pending count are 0 before it actually clears the view;